drivers/gpio/gpio_cc23x0.c

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) 2024 Texas Instruments Incorporated
+ * Copyright (c) 2024 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_cc23x0_gpio
+
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/irq.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/gpio/gpio_utils.h>
+
+#include <driverlib/clkctl.h>
+#include <driverlib/gpio.h>
+#include <inc/hw_ioc.h>
+
+#define IOC_ADDR(index)       (IOC_BASE + IOC_O_IOC0 + (sizeof(uint32_t) * (index)))
+#define GPIO_PIN_TO_MASK(pin) (1 << (pin))
```

Comment:
there's already a `BIT` macro that does this

Reply:
Thanks , fixed